metadata-ingestion/setup.py

```diff
@@ -143,6 +143,12 @@ def get_long_description():
     "pyspark==3.0.3",
```

Comment:
Given that this is under `framework_common`, is this Spark version a requirement of datahub? I'm not super familiar with datahub, but I am familiar with Spark and Iceberg.

Reply:
This dependency has nothing to do with Iceberg.  If I'm not mistaken, there is a source in Datahub that is using Spark to infer files schema and/or perform data profiling (stats gathering).